omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -190,7 +193,7 @@ def backend_verify_postgres_version(connection)
     # Note that we're looking for the same major, and using our minor as the minimum version
     # This provides compatibility with external databases that use < 9.6 before we make use
     # of any features available in > 9.2.
-    unless (major == REQUIRED_MAJOR) && (minor >= REQUIRED_MINOR)
+    unless SUPPORTED_MAJOR.include?(major) || ((major == REQUIRED_MAJOR) && (minor >= REQUIRED_MINOR))
```

Comment:
based on the comment above should we all only 13.3, since none of the other versions are tested.

Reply:
Done.